index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <title>Personal Portfolio</title>
+
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+  <body>
+    <section id="navbar">
+      <ul>
+        <li><a href="#welcome-section">Home</a></li>
+        <li><a href="#projects">Projects</a></li>
+        <li><a href="#contact">Contact Me</a></li>
+        <!-- <li><a href="#welcome-section">Home</a></li> -->
+      </ul>
+    </section>
+    <section id="welcome-section">
+      <div class="container welcome-text">
+        <h1>Hello, you are welcome!</h1>
+      </div>
+    </section>
+    <section id="projects">
+      <div class="project-title">
+        <!-- <img src="./" alt="" /> -->
```

Comment:
Nice work😊, 
I think removing unwanted code comments will make the work professional.